scripts/imgtool/__init__.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-imgtool_version = "1.6.0"
+imgtool_version = "1.7.0"
```

Comment:
Are we releasing MCUBoot `1.7.0`?

Reply:
I don't know about any plan to release MCUBoot. Suppose if backward compatibility wasn't broken it will be 1.7.0.

Reply:
No, we are not releasing `1.7.0` so this can't be `1.7.0`... update it to `1.7.0a1`!